src/doc/man/generated_txt/cargo-yank.txt

```diff
@@ -4,7 +4,8 @@ NAME
        cargo-yank - Remove a pushed crate from the index
 
 SYNOPSIS
-       cargo yank [options] --version version [crate]
+       cargo yank [options] crate@version cargo yank [options] --version
```

Comment:
This looks strange with both examples on the same line.

Reply:
Missed the `\` at the end; fixed now.